multimedia/ffmpeg/patches/030-h264-mips.patch

```diff
@@ -0,0 +1,18 @@
+--- a/libavcodec/mips/cabac.h
++++ b/libavcodec/mips/cabac.h
```

Comment:
This patch is missing details from Git, I mean who is the author of this patch, what is the commit subject, description, etc.

Reply:
This was me. Details are fuzzy but this is either a MIPS16 compilation fix, a glibc compilation fix, or a generic MIPS one.

Reply:
Okay, then please lets correct the mistakes from the past and include what is necessary. Some users might complain that the patch can not be applied, because of the missing details.